package.json

```diff
@@ -24,13 +24,14 @@
     "json-e": "^4.7.0",
     "material-ui-popup-state": "^5.1.2",
     "moize": "^6.1.6",
+    "moment-timezone": "^0.5.46",
```

Comment:
We don't really want to add `moment` as we already have `dayjs` :-) `moment` is a huge library.

Also nowadays the web platform already has this functionality, you can read about it on MDN => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString
It's possible we can replace `dayjs` in the process too.

Reply:
Initially I used day.js with timezone plugin to handle time zones, but the output format is limited to either IANA timezones (America/New_York) or the GMT offset (GMT+/-X), as day.js doesn’t provide abbreviations like PST or EST, so I added moment (as to make it look like the screenshot Carla has sent above, you can also see how it was before when I used dayjs (in the second screenshot she shared)). Let me know how you’d prefer me to proceed!

Reply:
![Image 11-10-2024 at 7 49 PM](https://github.com/user-attachments/assets/dcfd018f-2c17-49e2-a105-517711da20ca)
This is also what I can get with day.js. Would this be better and compact (compared to IANA timezones) so I can make the changes accordingly?

Reply:
I have made the changes! Please review them when possible

Reply:
Can you remove this change from package.json and run npm i again?